data/db_demo.yaml

```diff
@@ -30,9 +47,13 @@ groups/=program_A/users:
   - username: fulluser@cesium-ml.org
     admin: false
 
+groups/=program_A/streams:
+  - stream_id: =ztf_partnership
+
 filters:
-  - query_string: sample_query_string
+  - name: Yellow transients
     group_id: =program_A
+    stream_id: =ztf_partnership
```

Comment:
`stream_ids`?

Reply:
Ah, I see what you mean.. To my taste, it's easer to define, e.g. the Partnership stream, as Public-only+Partnership-only data as opposed to Partnership-only, so that a filter is associated with a single stream (and a single group) -- this also simplifies the book-keeping on the K side a bit. After all, alerts with programid=2 come with programid=1 stuff in prv_candidates.

Reply:
This makes me think - are we going to allow streams with data from multiple instruments? E.g., a ZTF PID=2  + Gattini stream?

Reply:
In my opinion, no, it would significantly complicate the alert filtering logic (most certainly on the K side). 
This is what the groups are for (among other things): they may have access to multiple streams from different discovery engines and you would define dedicated filters on each stream.